src/main/java/dk/cngroup/lentils/entity/User.java

```diff
@@ -41,7 +29,7 @@ public User() {
     @Column(name = "password", nullable = false)
     private String password;
 
-    @ManyToMany(cascade = CascadeType.DETACH, fetch = FetchType.EAGER)
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Nechtel si to umazat u Cypher?

Reply:
Máš pravdu, to jsem se hodně překlikl :zipper_mouth_face:

Reply:
V pohodě. Ono je možné, že to není úplně dobře ani tady. Ale radši bych to zatím nechal.